app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,66 @@
+class OrdersController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_order, only: %i[show edit update destroy]
+
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @orders = Order.find(params[:id])
+  end
+
+  def edit
+    @order = Order.find(params[:id])
+  end
+
+  def new
+    @order = Order.new
+  end
+
+  def create
+    # @order = Order.new(product_name: '...', product_count: '...') # no filtering and security checks this way
+    @order = Order.new(order_params)
+    if @order.save
+      flash.notice = 'The order was created successfully'
+      # redirect_to @order
+      render :new
```

Comment:
@bbcc33 it looks like you forgot to put this back after the troubleshooting we were doing. We should redirect to order here. No need to resubmit the PR, just FYI.